conan/tools/meson/toolchain.py

```diff
@@ -420,12 +420,14 @@ def _get_env_list(v):
     def _filter_list_empty_fields(v):
         return list(filter(bool, v))
 
+    @staticmethod
+    def _sanitize_format(v):
+        if v is None or isinstance(v, list):
+            return v
+        ret = [x.strip() for x in v.split() if x]
+        return f"{ret[0]}" if len(ret) == 1 else ret
```

Comment:
Let's rename this to something more specific:

```suggestion
    @staticmethod
    def _sanitize_env_format(value)
        if value is None or isinstance(value, list):
            return value
        ret = [x.strip() for x in value.split() if x]
        return ret[0] if len(ret) == 1 else ret
```